### DISCUSSION TITLE
建议添加说说显示相应标签分类和说说标签保存的功能

### DISCUSSION THREAD

**Discussion by Moriefy** [Qexo/Qexo#587] [Ideas]
尊敬的开发者，
您好！我在使用Qexo这个卓越的静态博客平台时，发现了一个可能需要改进的地方。目前，我的博客包含多个不同的栏目，比如生活日志和技术日志，我希望能有一个功能，可以按照不同的标签来分类显示我的说说。然而，我注意到Qexo似乎还没有这样的功能，如果能够添加这个功能，我相信它将极大地提升用户体验。
此外，我还想提出一个小建议，即在说说的标签功能中加入一个保存按钮。目前每次输入标签名称时都需要重新输入，这确实有些不便。如果能够实现标签的保存功能，那么不仅能够简化操作流程，也能让上述提到的按标签分类显示说说的功能更加方便实用。
我对您编写并开源了Qexo这一杰出项目表示衷心的感谢。它为像我这样的静态博客爱好者提供了极大的便利。如果能够考虑并实现上述建议，我相信Qexo将变得更加完善和易于使用。再次感谢您的辛勤工作和对社区的贡献，期待Qexo未来能够带来更多的惊喜。
衷心感谢您的考虑和支持。

**Comment by am-abudu** [comment]
可以考虑在提交说说之后在浏览器端自动保存相关数据。你觉得这个逻辑如何

**Reply by Moriefy** [reply]
可以的😁
